frontend/src/scenes/experiments/ExperimentView/ExperimentView.tsx

```diff
@@ -41,10 +44,16 @@ const ResultsTab = (): JSX.Element => {
         hasMinimumExposureForResults,
         statsMethod,
     } = useValues(experimentLogic)
+    /**
+     * we still use the legacy metric results here. Frequentist results are loaded
```

Comment:
nit: This will also be the case bayesian soon
```suggestion
     * we still use the legacy metric results here. Results on the new format are loaded
```